packages/ui/react-ui/src/CrossMintButtonStatus.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { CSSProperties, FC, MouseEvent, MouseEventHandler, useMemo, useCallback } from "react";
+import useCrossMintStatus, { OnboardingRequestStatusResponse } from "./hooks/useCrossMintStatus";
+import { Button, Img, Paragraph } from "./styles/index";
+
+export interface StatusButtonProps {
+    className?: string;
+    disabled?: boolean;
+    onClick?: (e: MouseEvent<HTMLButtonElement>) => void;
+    style?: CSSProperties;
+    tabIndex?: number;
+    clientId: string;
+    auctionId: string;
+    theme?: "light" | "dark";
+}
+
+export const CrossMintStatusButton: FC<StatusButtonProps> = ({
+    className,
+    disabled,
+    onClick,
+    style,
+    tabIndex,
+    theme = "dark",
+    clientId,
+    auctionId,
+    ...props
+}) => {
+    const status = useCrossMintStatus({ clientId });
+
+    const handleClick: MouseEventHandler<HTMLButtonElement> = useCallback(
+        (event) => {
+            if (onClick) onClick(event);
+
+            if (status === OnboardingRequestStatusResponse.WAITING_SUBMISSION) {
+                window.open(
+                    `https://crossmint.io/developers/onboarding${clientId ? `?clientId=${clientId}` : ""}${
```

Comment:
Now that we're on it can we change this url to start with www. ?

crossmint.io redirects to www.crossmint.io

Reply:
How about I change them all in https://github.com/CrossMint/crossmint-client-sdk/pull/43 since Im using an enum there?

Reply:
ok